tests/cupy_tests/math_tests/test_sumprod.py

```diff
@@ -1037,3 +1037,58 @@ def test_gradient_bool_input(self):
             x = testing.shaped_random(shape, xp, dtype=numpy.bool_)
             with pytest.raises(TypeError):
                 xp.gradient(x)
+
+
+@testing.gpu
```

Comment:
For a historical reason, we no longer need `@testing.gpu` decorator. Would you remove it?